src/reaper_osara.cpp

```diff
@@ -3583,6 +3583,26 @@ void cmdUnselAllTracksItemsPoints(Command* command) {
 		outputMessage(translate("unselected tracks/items/envelope points"));
 }
 
+void reportTempoTimeSig() {
+ReaProject* proj=EnumProjects(-1, nullptr, 0);
+if(proj==nullptr) return;
+double tempo=0;
+int timesig1=0;
+int timesig2=0;
+double pos=GetPlayPosition();
+TimeMap_GetTimeSigAtTime(proj, pos, &timesig1, &timesig2, &tempo);
+outputMessage(formatDouble(tempo, 1, false)+", "+to_string(timesig1)+"/"+to_string(timesig2));
```

Comment:
Rather than concatenating strings like this, it would be cleaner and easier to follow to use format(). This way, it's much easier to see what the output will look like. Something like this:

```
outputMessage(format("{}, {}/{}",
	formatDouble(tempo, 1, false), timesig1, timesig2));
```

You could even name the format template arguments, though that might be overkill for only three arguments for an untranslatable message:

```
outputMessage(format("{tempo}, {num}/{denom}",
	"tempo"_a=formatDouble(tempo, 1, false),
	"num"_a=timesig1, "denom"_a=timesig2));
```

I could go either way.